### ISSUE TITLE
Favicon download button gone

### ISSUE THREAD

**Issue by eickeh** [keepassxreboot/keepassxc#10790]
**First of all, thanks for your great work!**

## Overview
In previous versions, there was  a favicon download button next to the URL of the "Edit entry" view. After updating to version 2.7.7 it's gone and I also can't see the "use DuckDuckGo for Favicon download" setting anymore. Was this removed? 😱 Or am I just missing something? That was one of these tiny features that make me really love KeepassXC.

## Steps to Reproduce
1. Create or edit an entry. 
2. Favicon download button is not there

## Expected Behavior
Have an favicon download button

## Actual Behavior
Favicon download button is gone

## Info
KeePassXC - Version 2.7.7
Revision: 68e2dd8

[NOTE]: # ( Pick choices based on your environment )
Operating System: Linux (Debian Trixie)
Desktop Env: Gnome
Windowing System: X11

![image](https://github.com/keepassxreboot/keepassxc/assets/33962585/7d06ae52-5fe4-4925-bb45-93af24af674e)

**Comment by granite** [comment]
> In previous versions, there was a favicon download button next to the URL of the "Edit entry" view. After updating to version 2.7.7 it's gone

I noticed that also, but not sure what version it changed. I have KeePassXC 2.7.8, and the download favicon button has been moved to the Icon section.  If you click on the Icon section, you will see the Download favicon button.

>  I also can't see the "use DuckDuckGo for Favicon download" setting anymore. 

Settings->Application Settings->Security->Privacy->Use DuckDuckGo service to download website icons

**Comment by phoerious** [comment]
The location of the button as not changed. Are you running Debian? If so, please report the issue to them or switch to using our PPA or Flatpak. See #10725

**Comment by eickeh** [comment]
Ah, thanks @phoerious for the pointer, I indeed run Debian (added this to my first post now). 
@granite: I checked the icons tab before posting but it's not there either. As I run the version of the Debian Repositories, the functionality was not even compiled in (see #10725) and is thus missing. 

Sorry devs for creating another issue on this. It might save some issues when a big fat notice like this is included in the issue template:
`Debian is shipped with a minimal version of keepassxc now by default. Read https://github.com/keepassxreboot/keepassxc/issues/10725 before posting if you miss some functionality. `
